gencode.c

```diff
@@ -50,6 +50,10 @@
 
 #endif /* _WIN32 */
 
+#if defined(BSD) || (defined (__APPLE__) && defined(__MACH__))
+#include <net/bpf.h>
```

Comment:
So why does this need to be included?  We're dealing with *raw* filters, i.e. raw BPF machine code, so there shouldn't be any need to look at the definitions for the kernel's BPF interpreter.